docs/source/Enum-Mapping.md

```diff
@@ -41,15 +41,43 @@ internal class YourProfile : Profile
     ...
 ```
 
+### Mapping Enum's names
```

Comment:
That's too specific to be useful. Generally speaking you don't need the `ToString` suffix, you just need a `string` property and perhaps a `MapFrom`. We don't need this paragraph.

Reply:
`you just need a string property` is not correct in the case when I need to map both to int value and string value, because I can't have `string Value` and `int Value` properties in one object.

Reply:
Again, not useful.

Reply:
Useful to me, I needed that, and I couldn't find anything about that in the docs and ha to figure it out by random trial and error...
Here https://docs.automapper.org/en/stable/Queryable-Extensions.html#string-conversion it is mentioned, that ToString() method is automatically added to to each non-string to string mapping. But nowhere it is said, that I can write ToString in a property name, to trigger this method.